content/chainguard/chainguard-registry/artifactory-pull-through.md

```diff
@@ -0,0 +1,164 @@
+
+This command will return a `docker login` command like the following.
+
+```
+. . .
+
+To use this pull token in another environment, run this command:
+
+	docker login "cgr.dev" --username "<pull_token_ID>" --password "<password>"
+```
+
+Take note of the value that appears in place of `<password>` as you'll need this when you configure a new remote Artifactory repository for pulling through Production Images.
+
+After noting your `<password>` value, you can begin setting up an Artifactory repository from which you can pull Chainguard Production Images. As mentioned previously, this process is similar to the one outlined previously for setting up an Artifactory repository. 
+
+Create another repository and again select the **Remote** option. Also, be sure to once again choose **Docker** as the type of remote repository you want to set up. Enter the following details for your new remote repository in the **Basic** configuration tab:
+
+* **Repository Key** — Again, you can choose whatever name you like here, but this guide's examples will use the name `cgr-private`.
+* **URL** — This must be set to `https://cgr.dev/`.
+* **Password / Access Token** — Enter the `<password>` value you noted down from the `docker login` command you received 
+* **Include Patterns** — Ensure that you use the default value (`**/*`) in this field.
+
+
+![Screenshot of Artifactory's New Remote Repository "Basic" Configuration tab. The "Repository Key" field is set to "cgr-private", the "URL" field is set to "https://cgr.dev/", the username is set to "inky@example.com", the "Include Patterns" section has only the default entry "**/*", and the "Password / Access Token" field is set to a long hidden password.]()
+
+Lastly, in the **Advanced** configuration tab, ensure that the **Block Mismatching Mime Types** option is checked. Again, this should be the default.
+
+
+
+![Screenshot of Artifactory's New Remote Repository "Advanced" Configuration tab. This example shows all the default settings, including that the "Block Mismatching Mime Types" box is checked.]()
+
+Following that, click the **Create Remote Repository** button. If everything worked as expected, a modal window will appear letting you know that the repository was created successfully. You can click the **Set Up Docker Client** button at the bottom of this window to retrieve the commands you'll use to test that you can pull Images through this repository.
+
+### Testing pull through of a Chainguard Developer image: 
+
+After clicking the **Set Up Docker Client** button, a modal window will appear from the right side of the page. Click the **Generate Token & Create Instructions** button, which will generate two code blocks.
+
+The first will be a `docker login` command similar to the following example. Copy this command and run it in your terminal.
+
+```sh
+docker login -uinky@chainguard.dev myproject.jfrog.io
```

Comment:
Do you want to use carrots for the variables here too? `<myproject>.jfrog.io`? I just wonder about the consistency.

Reply:
yes that makes sense.